docs/pages/components/table.md

```diff
@@ -163,40 +163,40 @@ A table is a set of tabular data. Line items can support data, images and action
 
 ## Table with checkbox
 The checkbox input can be used at the beginning of each row to allow for bulk actions.
-It is recommended to add the parameter `area-selected="true` to the row that is selected.
+It is recommended to add the parameter `aria-selected="true` to the row that is selected.
 
 {% capture table-checkbox %}
 <table class="fd-table">
-    <thead>
-        <tr>
-            <th scope="col"><input type="checkbox"></th>
-            <th scope="col">Column Header</th>
-            <th scope="col">Column Header</th>
-            <th scope="col">Column Header</th>
-            <th scope="col">Column Header</th>
+    <thead class="fd-table__header">
+        <tr class="fd-table__row>
```

Comment:
Just a small typo here - missing the `"` and then I think we're ready to 🚢 🎉

Reply:
I pushed the changes to this quick as I was reviewing it and I agree with @jbadan -- this looks good.